### ISSUE TITLE
Failing to open .bam files

### ISSUE THREAD

**Issue by tristanpwdennis** [ANGSD/angsd#196]
Hi, 

I am trying to call GLs on some data using the command:

```./apps/angsd/angsd -b infile.bam -out outfile -dosaf 1 -GL 1 -doGlf 3 -doMaf 1 -doMajorMinor 1 -doPost 1 -doVcf 1 -doCounts 1 -doDepth 1 -dumpCounts 1 -doIBS 1 -makematrix 1 -doCov 1 -P 7 -SNP_pval 1e-6 -minMapQ 20 -minInd 5 -minQ 20 -minMaf 0.05 -doMajorMinor 1 >& angsd_outfile.log```

Every time I try, with a list of files, or individually, the program aborts with the message:

[E::hts_open_format] Failed to open file ?
angsd: multiReader.cpp:348: multiReader::multiReader(int, char**): Assertion `in' failed.
Aborted (core dumped)

I have tried with different .bam files, and I keep getting this error.

Any help would be really appreciated - thank you!

**Comment by ANGSD** [comment]
change -b to -i

-b expect a filelist containing all your bams or crams.
-i expects a single file.

Best

Den ons. 23. jan. 2019 kl. 13.48 skrev Tristan Dennis <
notifications@github.com>:

**Comment by tristanpwdennis** [comment]
Thanks for the speedy reply. This works for single files

I'm now trying with a list of files with -b and a .txt of my .bams and I'm hitting the same issue.

```[E::hts_open_format] Failed to open file bam1.bam
angsd: multiReader.cpp:348: multiReader::multiReader(int, char**): Assertion `in' failed.
Aborted (core dumped)```

The open failure is on a file that opened fine individually.

Thanks!

**Comment by tristanpwdennis** [comment]
UPDATE: This seems to have been fixed by adding the path/to/the/bam.bam files in the list .txt file
Thanks for your help :)

**Comment by dlema82** [comment]
Hello I am encountering the same issue. I edited my script to include path/to/bam.filelist as well as including path/to/files.bam 
I get the error
`        -> angsd version: 0.930 (htslib: 1.9) build(Nov  6 2019 21:46:42)
" : No such file or directoryo open file "/array/dlema/A34746_mtgenome.mapped.bam
angsd: multiReader.cpp:380: multiReader::multiReader(int, char**): Assertion `in' failed.
Aborted (core dumped)
`
But I'm sure the file is there in that path. I checked for typos and for unix line endings (as opposed to dos), but I keep getting this eroor that it can't find or open the .bam files
